pwa/src/components/handlers/handlerTable.tsx

```diff
@@ -36,7 +36,7 @@ export default function HandlersTable({ endpointId }) {
   };
 
   const handleSetDocumentation = (): void => {
-    API.Documentation.get("attributes")
+    API.Documentation.get("handlers")
```

Comment:
This is not returning any content.

(Might be a known issue)

Reply:
handlers doesn't have content yet